### PR TITLE
[hPaeYrJX] Fail apoc.warmup.run on non record storage dbs

### PR DESCRIPTION
**Pull request by loveleif**

### CODE REVIEW

File: it/src/test/java/apoc/it/core/WarmupEnterpriseTest.java [hunk 1]
```diff
@@ -20,51 +20,81 @@
...
-
-        session.close();
-        neo4jContainer.close();
+        final var conf = Map.of(
+                GraphDatabaseInternalSettings.include_versions_under_development.name(), "true",
+                GraphDatabaseSettings.db_format.name(), "multiversion");
+        withSession(conf, session -> {
+            RuntimeException e =
+                    assertThrows(RuntimeException.class, () -> testCall(session, "CALL apoc.warmup.run()", (r) -> {}));
+            assertTrue(e.getMessage().contains("Record engine type unsupported"));
```
Comment (gem-neo4j) [inline it/src/test/java/apoc/it/core/WarmupEnterpriseTest.java:51]: Didn't you delete this message? 🤔

Comment (loveleif) [reply it/src/test/java/apoc/it/core/WarmupEnterpriseTest.java:51]: Nice catch! I did, I assume this test is not run as part of the github build? 🤔🤔🤔

Comment (loveleif) [reply it/src/test/java/apoc/it/core/WarmupEnterpriseTest.java]: fbf1a8a02baccb87a58c76a621b9d47f0aab3986

**Review by gem-neo4j — APPROVED** [review]
Great!

**Pull request merged**